pkg/autopilot/checks/removedapis.go

```diff
@@ -23,10 +23,14 @@ import (
 
 type removedAPI struct {
 	group, version, kind, removedInVersion string
+	// currentVersion declares a version that is still supported for the Group Kind.
+	// If it's empty, it means that the Group Kind is removed in the removedInVersion.
+	currentVersion string
```

Comment:
Since the both return values mention version, but different ones, probably makes sense to mention it in names, like `removedInK8sVersion` and `currentAPIVersion` or smth

Reply:
Changed it.